src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -5020,7 +5024,7 @@ void Goo()
         Bar([|(1, ""hello"")|]);
         Bar((1, ""hello"");
     }
-}", count: 2);
+}", count: 3);
         }
```

Comment:
i don't see any tests validating the new behavior at all.

Reply:
same applies for VB>

Reply:
They're at https://github.com/dotnet/roslyn/pull/45918/files#diff-4750f840d4900df0c3b4fe6a6131e6b3R251 and https://github.com/dotnet/roslyn/pull/45918/files#diff-4750f840d4900df0c3b4fe6a6131e6b3R219 for C#, is there something wrong with them?

Reply:
ah, you have them in the interactive-tests file (i.e. for C# interactive/csx).  Can you move into the non-interactive tests?